components/Buttons/examples/ButtonsStoryboardAndProgrammatic.swift

```diff
@@ -1,12 +1,12 @@
 /*
  Copyright 2016-present the Material Components for iOS authors. All Rights Reserved.
-
+ 
```

Comment:
Please revert the changes to the copyright notice. If you like, you can make the changes in a follow-up, though it's not necessary. :)